README.md

```diff
@@ -21,22 +21,23 @@ Luos provides a simple way to think your hardware products as a group of indepen
 ## The most for the community​
 Most of the embedded developments are made from scratch. By using Luos, you will be able to capitalize on the development you, your company, or the Luos community already did. The re-usability of features encapsulated in Luos services will fasten the time your products reach the market and reassure the robustness and the universality of your applications.
 
-* → Join the [Luos Slack Community](http://bit.ly/JoinLuosSlack)
-* → Join the [Luos Discord Community](http://bit.ly/JoinLuosDiscord)
-* → Join the [Luos Reddit Community](http://bit.ly/JoinLuosReddit)
-* → Join the [Luos Forum](http://bit.ly/JoinLuosForum)
+* → Join the [Luos Discord Community](https://discord.gg/luos)
+* → Join the [Luos Reddit Community](https://www.reddit.com/r/Luos/)
+* → Follow us on [Twitter](https://twitter.com/Luos_io)
+* → Follow us on [Linkedin](https://www.linkedin.com/company/luos)
+* → Watch our new videos on [Youtube](https://www.youtube.com/channel/UCWeIoHVY9Z-04kdwXNtv2FA)
 
-Need dedicated help with your project? Check out [Luos' support packages](https://discord.com/invite/luos).
+Need dedicated help with your project? Check out [Luos' support packages](https://discord.gg/luos).
 
 ## Good practices with Luos​
 Luos proposes organized and effective development practices, guaranteeing development flexibility and evolutivity of your hardware product, from the idea to the maintenance of the industrialized product fleet.
 
 ## Let's do this​
 This section details the features of Luos technology as an embedded development platform, following these subjects:
 
-* Let's test through the [Luos get started](https://docs.luos.io/tutorials/get-started), to build, flash, run, and control your very first Luos code.
-* The [Basics of Luos](https://docs.luos.io/docs/luos-technology/basics/), explaining the general concepts and the project organization.
-* Definition of [Nodes](https://docs.luos.io/docs/luos-technology/node/), and the relation between Luos and the physical world.
-* Definition of [Packages](https://docs.luos.io/docs/luos-technology/package/), and how to make a portable and reusable development.
-* Definition of [Services](https://docs.luos.io/docs/luos-technology/services/), how to create and declare features in your product.
-* Definition of [Messages](https://docs.luos.io/docs/luos-technology/message/), when, why, and how to handle them, explaining the more advanced features of Luos.
+* Let's test through the [Luos get started](https://www.luos.io/tutorials/get-started), to build, flash, run, and control your very first Luos code.
```

Comment:
Remove trailing slash at the end of our URLs to avoid 308 redirects

Reply:
Done